Doc/tools/extensions/pyspecific.py

```diff
@@ -58,11 +59,33 @@
 
 def issue_role(typ, rawtext, text, lineno, inliner, options={}, content=[]):
     issue = utils.unescape(text)
+    # sanity check: there are no bpo issues within these two values
+    if int(issue) > 47261 and int(issue) < 400000:
+        msg = inliner.reporter.error(f'The bpo id {text!r} seems too high -- '
+                                     f'use :gh:`...` for GitHub ids.', line=lineno)
```

Comment:
```suggestion
        msg = inliner.reporter.error(f'The BPO ID {text!r} seems too high -- '
                                     'use :gh:`...` for GitHub IDs', line=lineno)
```

Reply:
The newcomers who may think in GitHub-centric terms of `:issue:` and `:pr:` will have no idea what the BPO is. So I'd extend @hugovk's proposal:

```suggestion
        msg = inliner.reporter.error(f'The bugs.python.org issue ID {text!r} seems too high -- '
                                     'use :gh:`...` for GitHub issue and pull request IDs.', line=lineno)
```

Reply:
The newcomers who may think in GitHub-centric terms of `:issue:` and `:pr:` will have no idea what the BPO is. So I'd extend @hugovk's proposal:

```suggestion
        msg = inliner.reporter.error(f'The bugs.python.org issue ID {text!r} seems too high -- '
                                     'use :gh:`...` for GitHub issue IDs.', line=lineno)
```

Reply:
I think the message is clear enough -- even if they haven't met bpo elsewhere they will know they need to use :gh: instead.